pkg/kubelet/kuberuntime/kuberuntime_container_linux.go

```diff
@@ -244,6 +252,9 @@ func (m *kubeGenericRuntimeManager) calculateLinuxResources(cpuRequest, cpuLimit
 		cpuQuota := milliCPUToQuota(cpuLimit.MilliValue(), cpuPeriod)
 		resources.CpuQuota = cpuQuota
 		resources.CpuPeriod = cpuPeriod
+		if !enforceCpuLimit {
+			resources.CpuQuota = int64(-1)
+		}
```

Comment:
please move the code like this (grouping quota settings together)
```
resources.CpuQuota = cpuQuota
if !enforceCpuLimit {
		resources.CpuQuota = int64(-1)
}
resources.CpuPeriod = cpuPeriod
```
(quite) minor: we may want to flip the semantic of the new flag so we use positive conditions:
```
func (m *kubeGenericRuntimeManager) calculateLinuxResources(cpuRequest, cpuLimit, memoryLimit *resource.Quantity, disableCPUQuota bool) *runtimeapi.LinuxContainerResources {
     ...
    if disableCPUQuota {
        resources.CpuQuota = int64(-1)
    }
    ...
}
```

Reply:
done